include/WebServer.hpp

```diff
@@ -1,7 +1,7 @@
 #ifndef WEBSERVER_HPP
 #define WEBSERVER_HPP
 
-#define CONNECTION_TIMEOUT	180000
+#define CONNECTION_TIMEOUT	18000
```

Comment:
this still seems high, but we can fight about it later

Reply:
its 3 minutes, for the poll. if its not high enough, poll will timeout and the execution stops. That means for example if the poll has no traffic in 3 minutes, the web server shuts down.